src/index.html

```diff
@@ -1,13 +1,74 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
         content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+      >
+  </head>
+  <body>
+    <div
+      data-qa="card"
+      class="card"
+      >
+      <div class="card__logo">
+        <img
+        class="card__product"
+        src="images/imac.jpeg"
+        alt="APPLE A1419 iMac 27 Retina 5K Monoblock"
+        >
+      </div>
+      <p class="card__content">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</p>
+      <div class="card__code">
+        <p>Product code: 195434</p>
+      </div>
+      <div class="stars stars--4">
+        <div class="stars__block">
+          <div class="stars__star">
+          </div>
+
+          <div class="stars__star">
+          </div>
+
+          <div class="stars__star">
+          </div>
+
+          <div class="stars__star">
+          </div>
+
+          <div class="stars__star">
+          </div>
+
+        </div>
+        <div class="stars__content">
+          <p>Reviews: 5</p>
+        </div>
```

Comment:
1) the same with redundant wrapper
2) rename class
```suggestion
          <p class="card__reviews">Reviews: 5</p>
```

Reply:
why do we use block name card if our text is part of the stars block.
maybe correct use stars__reviews ?

Reply:
only stars are part of `stars` block, not a `Reviews: 5`